src/status_im/chat/subs.cljs

```diff
@@ -375,8 +328,14 @@
 (reg-sub
  :get-last-message
  (fn [[_ chat-id]]
-   (subscribe [:get-ordered-chat-messages chat-id]))
- first)
+   (subscribe [:get-chat chat-id]))
+ (fn [{:keys [messages message-groups]}]
+   (->> (sort-message-groups message-groups messages)
+        first
+        second
```

Comment:
i would consider extracting `second first` pair to some function with more less meaningful name, if it's the same thing as here https://github.com/status-im/status-react/pull/4221/files#diff-50b2d2f85046da6681844adb5e140a06R91

Reply:
probably even `second first :message-id`